flowcraft-webapp/frontend/src/components/reports/phylogeny.js

```diff
@@ -40,17 +40,25 @@ export class Phylogeny extends React.Component{
 
         const metadata = new Map();
 
-        for (const m of rawMetadata){
-            const currentMeta = m.reportJson.metadata[0];
-            const sample = currentMeta.sample;
+        for (const m of rawMetadata) {
+            for (const x of m.reportJson.metadata) {
+
+                const currentMeta = x;
```

Comment:
Instead you could do `for (const currentMeta of m.reportJson.metadata) {` and remove this line

Reply:
Agreed! Thanks for the suggestion!